source/_integrations/nuki_matter.markdown

```diff
@@ -20,6 +20,17 @@ works_with:
 
 {% include integrations/wwha.md url="https://nuki.io" name="Nuki" %}
 
+## Setting up your Matter integration requires Thread
+
+To use this integration, you need a Thread border router that supports Matter. For more information, refer to the [Thread documentation](/integrations/thread/).
+
+## Home Assistant Connect ZBT-1 support
+
+The Home Assistant Connect ZBT-1 supports Matter devices over Thread and can function as a Thread border router for remote access. For information on setting up the Connect ZBT-1 as a Thread border router, refer to the [Thread documentation](/integrations/thread/).
```

Comment:
I'd remove the title of this section, and make this a second sentence of the above heading, e.g.:

You can turn your Home Assistant installation into a Thread border router e.g. by using the Home Assistant Connect ZBT-1. Refer to the [Turning Home Assistant into a Thread border router](/integrations/thread/#turning-home-assistant-into-a-thread-border-router) section of the Thread integration documentation